packages/renderer/src/lib/ui/NavPage.svelte

```diff
@@ -5,19 +5,19 @@ export let searchEnabled = true;
 </script>
 
 <div class="flex flex-col w-full h-full shadow-pageheader">
-  <div class="flex flex-col w-full h-full pt-4">
-    <div class="flex pb-2">
+  <div class="flex flex-col w-full h-full pt-4" role="region" aria-label="{title}">
+    <div class="flex pb-2" role="region" aria-label="header">
       <div class="px-5">
-        <h1 aria-label="{title}" class="text-xl first-letter:uppercase">{title}</h1>
+        <h1 class="text-xl first-letter:uppercase">{title}</h1>
       </div>
       <div class="flex flex-1 justify-end">
-        <div class="px-5">
+        <div class="px-5" role="group" aria-label="additionalActions">
           <slot name="additional-actions">&nbsp;</slot>
         </div>
       </div>
     </div>
     {#if searchEnabled}
-      <div class="flex flex-row pb-4">
+      <div class="flex flex-row pb-4" role="region" aria-label="search">
```

Comment:
I would say the role should go on the search input field ?

Reply:
I think this could trigger more general discussion. I think there are multiple options where to put `region` role when talking about HTML structure that consist of multiple nested divs. I can imagine keeping the role `region` somewhere at the one of the top divs that marks some area in the doc. In this case of search input, I guess, that is pretty straightforward, but what if we wanted to extend the area of the search part by, let's say, drop down element where to search more specifically (like status used vs. unused)? Then It would make sense to have a region in the page that covers multiple by nature aligned elements. That is also what `region` is the closest to - identify a part of doc. though I am no expert here and have not found better aria role (landmark one)...